### PR TITLE
Create strings.xml for South Korea

### PR DESCRIPTION
**Pull request by onestone9900**
Hi, I created string.xml for South Korea
I confirm PR history and test my smart phone before pull requset.
but if exist any wrong form, comment please~
Thank you!! :)

### CODE REVIEW

**Comment by naXa777** [comment]
If you don't mind, I'd like to import your translation in my fork of this repository. I've already created a pull request here https://github.com/naXa777/SoundRecorder/pull/41 (the app is in alpha testing; not released yet)

**Review by naXa777 — APPROVED** [review]

**Comment by onestone9900** [comment]
> If you don't mind, I'd like to import your translation in my fork of this repository. I've already created a pull request here [naXa777#41](https://github.com/naXa777/SoundRecorder/pull/41) (the app is in alpha testing; not released yet)

my pleasure :) !!!

**Comment by naXa777** [comment]
@sowen2819 I see that you added commits unrelated to this PR. Please, keep it clean: one PR - one change. It may contain multiple commits, but they should be related to the PR topic ("South Korean translation" in this case).

You probably did it unintentionally, because you do everything in `master` branch. In order to prevent such kind of errors in future, please work in a new branch, then create a pull request from this branch. Then go back to `master` and create a new branch if you want to implement anything else.
Also, if you want you can create PR to multiple repositories and/or merge the branch into your `master` branch.

I hope my suggestions will help you to keep your git history and PRs clean.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,8 @@
-Easy Sound Recorder
+Sosimhan Recorder
```
Comment (naXa777) [inline README.md]: Please keep this change local to your repository.

File: app/src/main/java/com/danielkim/soundrecorder/fragments/RecordFragment.java [hunk 2]
```diff
@@ -76,10 +74,19 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         View recordView = inflater.inflate(R.layout.fragment_record, container, false);
 
-        mChronometer = (Chronometer) recordView.findViewById(R.id.chronometer);
         //update recording prompt text
         mRecordingPrompt = (TextView) recordView.findViewById(R.id.recording_status_text);
 
+        mCountButton=(Button)recordView.findViewById(R.id.btnCount);
```
Comment (naXa777) [inline app/src/main/java/com/danielkim/soundrecorder/fragments/RecordFragment.java]: It's better to separate this change and create a new Pull Request. Please keep the existing PR clean and (maybe) it will be accepted by the repository owner (someday).

File: app/src/main/res/layout/fragment_record.xml [hunk 3]
```diff
@@ -16,19 +16,20 @@
         android:layout_centerHorizontal="true"
         android:src="@drawable/ic_mic_white_36dp" />
 
-    <Chronometer
+    <TextView
```
Comment (naXa777) [inline app/src/main/res/layout/fragment_record.xml]: This change is not related to the PR (it's not translation). Please revert it or create a new PR.

**Review by naXa777 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_record.xml (hunk 3 above)
Comment (sowen2819) [reply app/src/main/res/layout/fragment_record.xml]: oh.....my mistake.
So I reset my repository.
I thought sending PR only my previous commit( Create strings.xml for South Korea ). but continues to add commit in PR....
I'll be careful

thank you feedback. I try it!

**Comment by onestone9900** [comment]
> @sowen2819 I see that you added commits unrelated to this PR. Please, keep it clean: one PR - one change. It may contain multiple commits, but they should be related to the PR topic ("South Korean translation" in this case).
> 
> You probably did it unintentionally, because you do everything in `master` branch. In order to prevent such kind of errors in future, please work in a new branch, then create a pull request from this branch. Then go back to `master` and create a new branch if you want to implement anything else.
> Also, if you want you can create PR to multiple repositories and/or merge the branch into your `master` branch.
> 
> I hope my suggestions will help you to keep your git history and PRs clean.

oh.....my mistake.
So I reset my repository.
I thought sending PR only my previous commits( Create strings.xml for South Korea ). but continues to add commit in PR....
I'll be careful

thank you feedback. I try it!
